<?xml version="1.0"?>
<robot name="trinamic" xmlns:xacro="http://ros.org/wiki/xacro">

  <link name="world">

   </link>
 
   <joint name="base" type="fixed">
     <origin rpy="0 0 0" xyz="0 0 0"/>
     <parent link="world"/>
     <child link="shaft"/>
   </joint>

   <link name="shaft">

   </link>

   <joint name="fixed_joint" type="fixed">
     <origin rpy="0 0 0" xyz="0.0175 0 0.01"/>
     <parent link="shaft"/>
     <child link="shaft2"/>
   </joint>

  <link name="shaft2">

   </link>
 
   <joint name="servo1" type="continuous">
     <axis xyz="0 0 1"/>
     <origin rpy="0 0 0" xyz="0.02 0 0.01"/>
     <parent link="shaft2"/>
     <child link="ghost"/>
   </joint>
 
   <link name="ghost">
   
   </link>

   
 </robot>
